packages/infrastructure/mongodb/main.tf

```diff
@@ -0,0 +1,69 @@
+terraform {
```

Comment:
Better to call this folder for slackapp, as it's setting up the infrastructure needed for an slackapp.

Reply:
I like it better when the naming describes what it is not what it is a part of. It could be a part of many things.